bench_vectorizer.py

```diff
@@ -0,0 +1,47 @@
+from vectorizer import Vectorizer, VectorInput
```

Comment:
This file is useful for local testing, in order to run the vectorizing process independently from uvicorn.